### ISSUE TITLE
[Bug]: Focus Indicator for quiet sp-link not visible in Safari

### ISSUE THREAD

**Issue by spdev3000** [adobe/spectrum-web-components#5526]
### Code of conduct

- [x] I agree to follow this project's code of conduct.

### Impacted component(s)

Link

### Expected behavior

Focus indicator is shown for quiet links when tabbing over page and focussing sp-link.

### Actual behavior

Focus indicator is _not_ shown for quiet links when tabbing over page and focussing sp-link. 
Working fine in Chrome.

### Screenshots

Chrome:
![Image](https://github.com/user-attachments/assets/1f382d6b-29c9-4080-a141-9f87b7107d5f)

Safari:
![Image](https://github.com/user-attachments/assets/5b44f944-fd6c-48c3-ad74-dbb98fdc9a93)

### What browsers are you seeing the problem in?

Safari

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/link/#quiet-links
2. Tab over page (please notice that you might need to enable highlighting in general in Safari, see: https://corehelpcenter.bqe.com/hc/en-us/articles/115003481673-Tab-key-is-not-working-on-Mac-Safari) 
3. Focus indicator is not shown over quiet links in Safari

### Severity

SEV 4

### Would you like to track this issue in Jira?

- [ ] Yes, please tell me the ticket number!
